examples/jsm/lines/LineMaterial.js

```diff
@@ -7,7 +7,7 @@
  *  dashSize: <float>,
  *  dashOffset: <float>,
  *  gapSize: <float>,
- *  resolution: <Vector2>, // to be set by renderer
+ *  resolution: <Vector2>,
```

Comment:
This line should be removed.

@Mugen87 How do you feel about removing this entire comment block? Apparently, we no longer use this coding pattern.

Reply:
I'm fine with that.

Reply:
I'll fix it. It's time to move on.